### PR TITLE
fix missing metrics module

### PR DESCRIPTION
**Pull request by mumichae**
As a result of the refactoring, in a normal package install the metrics module is not included.

Here I want to fix this issue and add more tests for metrics

### CODE REVIEW

**Comment by ChineseBest** [comment]
Hi, are you trying to solve this problem?
ImportError: cannot import name 'metrics' from 'scIB' (/usr/local/lib/python3.7/dist-packages/scIB/__init__.py)

**Comment by mumichae** [comment]
> Hi, are you trying to solve this problem?
> ImportError: cannot import name 'metrics' from 'scIB' (/usr/local/lib/python3.7/dist-packages/scIB/**init**.py)

Yes, feel free to test if it works for you

**Comment by ChineseBest** [comment]
> > Hi, are you trying to solve this problem?
> > ImportError: cannot import name 'metrics' from 'scIB' (/usr/local/lib/python3.7/dist-packages/scIB/**init**.py)
> 
> Yes, feel free to test if it works for you

sorry, it seems that it still cannot work.
ImportError                               Traceback (most recent call last)
<ipython-input-2-c19642473bb1> in <module>()
----> 1 import scIB

1 frames
/usr/local/lib/python3.7/dist-packages/scIB/clustering.py in <module>()
      5 import scanpy as sc
      6 from scIB import utils
----> 7 from scIB import metrics
      8 
      9 

ImportError: cannot import name 'metrics' from 'scIB' (/usr/local/lib/python3.7/dist-packages/scIB/__init__.py)

**Comment by mumichae** [comment]
Did you make sure to remove and reinstall `scIB` before testing? Sometimes that is necessary for the changes to work.

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -23,14 +23,19 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install flake8 pytest
-        pip install -r requirements.txt
+        pip install .
```
Comment (LuckyMD) [inline .github/workflows/python-package.yml:26]: I would prefer explicit file handling...

**Review by LuckyMD — COMMENTED** [review]
Just 1 comment on requirements.txt explicit handling.

File: .github/workflows/python-package.yml (hunk 1 above)
Comment (mumichae) [reply .github/workflows/python-package.yml:26]: you mean reverting to the previous command? That way only the requirements are installed but not the package itself, so I wouldn't be able to test scIB import

Comment (ChineseBest) [reply .github/workflows/python-package.yml:26]: I am sure that I uninstall the scIB and try to reinstall it, but I still cannot import metric. You can also try this on colab by yourself.

**Comment by ChineseBest** [comment]
> pip install git+https://github.com/theislab/scib.git@fix_missing_metrics

ok, I can try this again.

**Comment by ChineseBest** [comment]
> Did you remove and reinstall `scIB` before testing? Sometimes that is necessary for the changes to work.
> Also make sure that you're installing the correct version ;)
> 
> ```
> pip install git+https://github.com/theislab/scib.git@fix_missing_metrics
> ```

that link works. Thanks a lot.

File: .github/workflows/python-package.yml (hunk 1 above)
Comment (LuckyMD) [reply .github/workflows/python-package.yml:26]: Ah, sorry... okay. I wasn't thinking clearly ^^.

**Review by LuckyMD — APPROVED** [review]

**Pull request merged**